src/EventuallyConsistentTestTrait.php

```diff
@@ -27,13 +27,16 @@
  */
 trait EventuallyConsistentTestTrait
 {
-    /* @var int The number of retries for eventually consistent tests */
-    protected static $eventuallyConsistentRetryCount = 3;
-
-    private function runEventuallyConsistentTest(
-        callable $func,
-        $retries = self::$eventuallyConsistentRetryCount
-    ) {
+    private function runEventuallyConsistentTest(callable $func, $retries = 0)
+    {
+        if ($retries <= 0) {
+            // It doesn't make sense, we'll use the default value. You can
+            // also define the static property $eventuallyConsistentRetryCount in
+            // your concrete class then we'll use that value.
+            $retries = isset(static::$eventuallyConsistentRetryCount)
```

Comment:
You can not modify the value of the static properties defined in trait, in your concrete class.
Well, we can have a default value like this:

```
trait EventuallyConsistentTestTrait
{
    protected static $eventuallyConsistentRetryCount = 3;
}
```

But since you can not modify the value of that property in your concrete class, you need to do something like:

```
$retries = isset(static::$_eventuallyConsistentRetryCount)
    ? static::$_eventuallyConsistentRetryCount
    : static::$eventuallyConsistentRetryCount;
```

which is ugly.